src/core/io/outputfb.h

```diff
@@ -0,0 +1,108 @@
+/*************************************************************************
+ * Copyright (c) 2025 Johannes Kepler University Linz
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *   Alois Zoitl - adds initial implementation
+ *************************************************************************/
+
+#pragma once
+
+#include "processinterfacefb.h"
+
+namespace forte {
```

Comment:
you could have `forte::core::io {` to remove some extra indentation (also in the other files)

Reply:
my compiler says this does not work.  I've also only seen it in using namespace statements.

Reply:
done